### PR TITLE
add xref to dementia

### LINKED ISSUE #6193: Add hpo xref to dementia
**Mondo term (ID and Label):**
MONDO:0001627 - dementia

**Xref that should be fixed (ID and label):**
HP:0000726 - Dementia

**Your nano-attribution (ORCID)**
0000-0001-9859-8589

**Other comments:**
xref url: https://hpo.jax.org/app/browse/term/HP:0000726
(MONDO:otherHierarchy)

### PR DESCRIPTION
**Pull request by nicolevasilevsky**
close #6193

### CODE REVIEW

**Comment by nicolevasilevsky** [comment]
this is minor, so I went ahead and merged
